mne_bids/read.py

```diff
@@ -190,6 +208,14 @@ def read_raw_bids(bids_fname, bids_root, verbose=True):
         Full name of the data file
     bids_root : str
         Path to root of the BIDS folder
+    read_params : dict | None
+        Additional parameters for reading BIDS compatible data in a dictionary.
+        The keys of the dictionary must correspond with potential input
+        parameters of MNE-Python's read_raw_* function for the data format you
+        intend to read. For example, when reading fif data you can specify
+        `read_params` as {allow_maxshield=False}. Non-fitting parameters will
```

Comment:
I wouldn't bother adding a new argument just for this. Let's do use-case driven software engineering. Ask people around you to try `mne-bids` and raise issues. It will give you a sense of gratification but also give a sense of priority.

Reply:
fully agreed. Also besides allow_maxshield all the other params contain info that should be present in the BIDS sidecar files.

Reply:
see my response: https://github.com/mne-tools/mne-bids/pull/244#issuecomment-514590193